### DISCUSSION TITLE
thresholds color should export to csv/excel

### DISCUSSION THREAD

**Discussion by hd-1803** [grafana/grafana#62629] [Dashboards, Panels & Visualizations]
currently export to csv [Download CSV] feature is not exporting thresholds color into excel/csv , this should show thresholds  color also in excel/csv

![image](https://user-images.githubusercontent.com/78032110/215861029-070d3a99-b683-4cd5-baa1-5387b184708a.png)
![image](https://user-images.githubusercontent.com/78032110/215861220-84d3a79a-7195-4405-b2c5-4aed3ab89395.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This will be highly dependent on the browser and what you're copying into - for instance, many plain text editors where you'd paste CSV values do not have the ability to show different colors. I would recommend setting up the color threshholds in the tool you paste values into.
